app/assets/javascripts/govuk-component/govspeak-enhance-youtube-video-links.js

```diff
@@ -26,20 +26,22 @@
 
   function enhanceYoutubeVideoLinks($el){
     $el.find("a[href*='youtube.com'], a[href*='youtu.be']").each(function(i){
-      var $link = $(this),
+      if (!$(this).attr("data-youtube-player") == "off") {
```

Comment:
Could this be extracted to a function for better readability? eg `shouldEnhance()`

It might be better to return early from the function, to avoid the extra nesting.